app/api/routes.py

```diff
@@ -244,7 +285,28 @@ def create_resource(json, db):
         paid=json.get('paid'),
         notes=json.get('notes'))
 
-    db.session.add(new_resource)
-    db.session.commit()
+    try:
+        db.session.add(new_resource)
+        db.session.commit()
+
+        return standardize_response(new_resource.serialize, None, "ok")
+    except Exception as e:
+        logger.error(e)
+        return standardize_response(None, [{"code": "bad-request"}], "bad request", 400)
```

Comment:
TODO: I would like to change these to `redirect('/400')` and put in a generic `400` handler. Also, might want to look into returning validation errors as `422` rather than `400`, and a generic exception here might actually need to be a 500...

This PR is growing and growing, and I don't want to keep growing it by adding all that in, so I'd rather roll with it as is and open those as separate issues that are beginner friendly